py3status/modules/volume_status.py

```diff
@@ -346,14 +343,16 @@ def on_click(self, event):
         """
         button = event['button']
         # volume up
-        if self.button_up and button == self.button_up:
+        if button == self.button_up:
             self.backend.volume_up(self.volume_delta)
         # volume down
-        elif self.button_down and button == self.button_down:
+        elif button == self.button_down:
             self.backend.volume_down(self.volume_delta)
         # toggle mute
-        elif self.button_mute and button == self.button_mute:
+        elif button == self.button_mute:
             self.backend.toggle_mute()
+        else:
+            self.py3.prevent_refresh()
```

Comment:
I'd not have this final else clause.

volume can be changed it many ways and so allowing the user to do a refresh click is useful.